modules/openfast-library/src/FAST_Lin.f90

```diff
@@ -3331,8 +3331,8 @@ SUBROUTINE Linear_ED_InputSolve_dy( p_FAST, y_FAST, SrvD, u_ED, y_ED, y_AD, u_AD
 
          ! ED translation displacement-to-ED moment transfer (dU^{ED}/dy^{ED}) from BD root-to-ED hub load transfer:
       ED_Start = Indx_u_ED_Hub_Start(u_ED, y_FAST) + u_ED%HubPtLoad%NNodes*3 ! start of u_ED%HubPtLoad%Moment field (skip forces)
+      ED_Out_Start  = Indx_y_ED_Hub_Start(y_ED, y_FAST) ! start of y_ED%HubMotion%TranslationDisp field
```

Comment:
Note that the documentation has this as a summation of all the terms, not terms at each ED root node to the hub. So this change makes it more consistent with the documentation and doesn't affect the resulting frequencies.

This also brings this closer in line with the tight coupling implementation.

Reply:
Based on the rest of the code, it looks like the original implementation where it connected to the blade root was a bug.... that apparently didn't affect the results. Thanks for fixing it!